specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/NetworkWatcherFlowLogCreate.json

```diff
@@ -0,0 +1,73 @@
+{
+  "parameters": {
+    "api-version": "2019-07-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "networkWatcherName": "nw1",
+    "flowLogName": "fl",
+    "parameters": {
+      "location": "centraluseuap",
+	    "properties": {
+	      "targetResourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityGroups/desmondcentral-nsg",
+        "storageId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/nwtest1mgvbfmqsigdxe",
+	      "enabled": true,
+	      "format": {
+		      "type": "JSON",
+		      "version": 1
+	      }
+	    }
+	  }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "Microsoft.Networkdesmond-rgdesmondcentral-nsg",
```

Comment:
`id` and `name` seem to be different. Shouldn't name be how its specified in parameters (`fl`)?